sphinx/source/conf.py

```diff
@@ -71,7 +71,7 @@
 
 autodoc_member_order = 'groupwise'
 
-bokeh_missing_google_api_key_ok = False
+bokeh_missing_google_api_key_ok = True
```

Comment:
This should always be set to `False` in the Bokeh repo proper. It is to ensure that automated builds without an API key set are impossible. Setting to True is only intended for users using `bokeh-sphinxext` in their own outside projects. FYI unless I specifically need to view maps, I usually locally build the docs like 
```
BOKEH_DOCS_CDN=local GOOGLE_API_KEY="junk" make serve
```

Reply:
Absolutely, I had it changed locally and accidentally included it in my commit. I have already reverted this change.